prism/src/main/java/org/apache/falcon/service/BacklogMetricEmitterService.java

```diff
@@ -206,6 +216,7 @@ public synchronized void onSuccess(WorkflowExecutionContext context) throws Falc
                 metrics.remove(new MetricInfo(DATE_FORMAT.get().format(date), context.getClusterName()));
                 if (metrics.isEmpty()) {
                     entityBacklogs.remove(entity);
+                    publishBacklog((Process) entity, context.getClusterName(), 0L);
```

Comment:
Why eager publish only for all instance succeed? Either due for all success instances or compute this in BacklogCalcService, so it is in sync with the frequency of publish.